docs/resize-event-handling-guide.md

```diff
@@ -0,0 +1,57 @@
+---
+id: resize-event-handling-guide
+title: Handling `resize` event
+sidebar_label: Handling `resize` event
+---
+
+Starting from version `0.5.1`, `react-slate` supports re-rendering app to the same target, like `process.stdout`, which means rendering after the terminal was resized is a matter of attaching `resize` event handler and calling `renderToTerminal` again:
+
+```js
+import { renderToTerminal } from 'react-slate';
+
+const App = () => {
+  /* ... */
+};
+
+// Initial render
+renderToTerminal(<App />, process.stdout);
+
+process.on('resize', () => {
+  // Re-render app with new terminal size
+  renderToTerminal(<App />, process.stdout);
+});
+```
+
+The above example works fine. However, you might feel lag when you resize terminal. That's because the `resize` event is fired multiple times and sometimes really often.
+
+The solution is to **throttle** or **debounce** the event handler. For that purpose we can use `lodash.throttle` or `lodash.debounce`:
```

Comment:
`lodash.*` are deprecated in favor of `lodash-es`/`lodash` with babel plugin according to it's author. So maybe just `lodash/debounce`?